packages/edit-site/src/components/sidebar-dataviews/default-views.js

```diff
@@ -49,7 +49,7 @@ const DEFAULT_PAGE_BASE = {
 export const DEFAULT_VIEWS = {
 	page: [
 		{
-			title: _x( 'All', 'pages' ),
+			title: _x( 'All pages', 'pages' ),
```

Comment:
```suggestion
			title: __( 'All pages' ),
```

Context may no longer be necessary.

Note: If you apply this suggestion, you will also need to remove `_x` from the import statement to pass the lint check.

Reply:
That's true, good suggestion :)